README.md

```diff
@@ -242,7 +242,7 @@ These dependencies are met by default by Nokogiri's packaged versions of the lib
 
 We provide native gems by pre-compiling libxml2 and libxslt (and potentially zlib and libiconv) and packaging them into the gem file. In this case, no compilation is necessary at installation time, which leads to faster and more reliable installation.
 
-See [`LICENSE-DEPENDENCIES.md`](LICENSE-DEPENDENCIES.md) for more information on which dependencies are provided in which native and source gems.
+See LICENSE-DEPENDENCIES.md for more information on which dependencies are provided in which native and source gems.
```

Comment:
Although rdoc will magically link this file, the nokogiri.org site is vanilla markdown and so we need to keep this as an explicit link so long as we're maintaining that site/toolchain.

```suggestion
See [LICENSE-DEPENDENCIES.md](LICENSE-DEPENDENCIES.md) for more information on which dependencies are provided in which native and source gems.
```

Reply:
Fixed.